cpu/lm4f120/include/cpu_conf.h

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (C) 2015 Rakendra Thapa <rakendrathapa@gmail.com
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @defgroup        cpu_lm4f120
+ * @ingroup         cpu
+ * @brief           CPU specific implementations for the Stellaris Launchpad LM4F120 board
+ * @{
+ *
+ * @file
+ * @brief           CPU specific hwtimer configuration options
+ *12
+ * @author          Rakendra Thapa <rakendrathapa@gmail.com>
+ */
+
+#ifndef __CPU_CONF_H
```

Comment:
here and all other files: don't pre-pend the file guards with underscored, as this is not allowed by the C-standard! So `s/__CPU_CONF_H/CPU_CONF_H/`